src/route.cr

```diff
@@ -0,0 +1,28 @@
+require "./access"
```

Comment:
I think this file should be namespaced under `DppmRestApi::Config`, and be at src/config/route.cr.